developer-docs/latest/cli/CLI.md

```diff
@@ -104,6 +104,7 @@ strapi configuration:dump
 
 Options:
   -f, --file <file>  Output file, default output is stdout
+  -p, --pretty       Format the output JSON with indentation and line breaks (default: false)
```

Comment:
Thanks for documenting this, I have one remark:
`(default: false)` Are boolean options (meaning optional) not always false? 😉🤔

Reply:
Hi Mattie - I agree - however, for consistency, I just copied what was already in the main code and visible when you do `strapi config:dump --help`. I would need to do two PRs to update this text.

Reply:
Aah okay, that explains. It's not that big of an problem 😉